### PR TITLE
Add Google Analytics

### PR DESCRIPTION
**Pull request by t-sommer**
This PR adds the Google Analytics tracking code (with IP anonymization enabled), a cookie consent banner and updates the privacy policy.
The cookie consent banner provides a link to a [website](https://cookiesandyou.com/) that explains in plain english how to remove any cookies and how to disable them altogether.
The website (fmi-standard.org) works fine even if the client rejects all cookies.
The updated privacy policy refers to detailed information on how the data is processed.

### CODE REVIEW

**Comment by chrbertsch** [comment]
We discussed at the last steering committee meeting which information to provide to listed tool vendors. 
The decision was to "stay neutral here and only provide access data of the website itself, no further data."

Does this mean, nobody (including webmaster) will get analytics data of individual tools due to current analytics implementation, or some people (for example, webmaster) will be able to see these data but they are not shared with listing tools?

**Comment by chrbertsch** [comment]
@mtiller , @sjoelund , @MartinOtter: please review this PR, it is now open for one month.

**Comment by andreas-junghanns** [comment]
Martin (O.): Why do we have to wait here? We are planning to invoice people for FMI tool listing starting April 1, 2019. And we need months to approve such a simple change that many people have analysed to be harmless.
If you need to discuss this with me before you can approve, please let me know when would be a good time and I will call. Otherwise, simply approve here and we can go on.

Bigger picture: This is part of the larger plan to make at least MAP FMI more efficient by providing the funds (through listing fees) to outsource tasks that we cannot find volunteers for. You can already see the benefits of the back office. We want more of these kinds of paid activities, that is what we need the funds for. Listing fees will come mostly from PR departments because it is the exposure that they are interested in. Google analytics is a way to measure the exposure - precisely what we are trying to market here. So this activity is a cornerstone to our overall strategy for more efficiency and impact as discussed and approved at the MA strategy meeting in Lund 18 months ago. I urge everybody to simply act and not re-question every step along the way after we have discussed and approved these issues at length multiple times.

**Comment by chrbertsch** [comment]
Torsten B.: Modelica web master Martin Sjölung seems to know what to do, he wants to do this together with Torsten S. I will sent the MA board meeting minutes to Torsten S.
